### PR TITLE
Mention Warning[]= that can be used to suppress the deprecation warnings

### PR DESCRIPTION
**Pull request by mame**

### CODE REVIEW

**Comment by ismailarilik** [comment]
I wonder why wasn't this PR emailed to me? I am "Watching" the repository.

**Comment by ismailarilik** [comment]
Sorry, it came but I have missed it somehow.

**Comment by stomar** [comment]
IMO the `<del>` section could be removed completely.

From the ChangeLog section it is obvious that a change was made, and the previous versions could be inspected in this repo. (But I do not think the changes of these warning messages during the preview/rc phase would be very relevant as soon as 2.7.0 is released,)

File: en/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md [hunk 1]
```diff
@@ -349,3 +355,7 @@ The automatic conversion not only confuses people but also makes the method less
 ## Acknowledgment
 
 This article was kindly reviewed (or even co-authored) by Jeremy Evans and Benoit Daloze.
+
+## ChangeLog
```
Comment (stomar) [inline en/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md]:
```suggestion
## History
```

File: en/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md [hunk 2]
```diff
@@ -349,3 +355,7 @@ The automatic conversion not only confuses people but also makes the method less
...
+
+* 23rd Dec. 2019: In 2.7.0-rc2, the warning message was slightly changed, and an API to suppress the warnings was added
```
Comment (stomar) [inline en/news/_posts/2019-12-12-separation-of-positional-and-keyword-arguments-in-ruby-3-0.md]:
```suggestion
* Updated 2019-12-23: In 2.7.0-rc2, the warning message was slightly changed, and an API to suppress the warnings was added.
```

**Comment by mame** [comment]
Thanks!  I've merged your suggestions.

**Comment by mame** [comment]
And removed `<del>`.

I'd like to merge this before Ruby 2.7.0 release (today :smirk:).

**Comment by ismailarilik** [comment]
@mame as I saw, some warnings didn't converted to their new forms. It will cause inconsistencies through the post. You should think to update them. You can find them by searching these sentences in the post source:

1. The last argument is used as keyword parameters
2. The keyword argument is passed as the last hash parameter
3. The last argument is split into positional and keyword parameters

**Pull request merged**